src/common/markdown.ts

```diff
@@ -43,16 +43,49 @@ const defaultTextRender = markdown.renderer.rules.text;
 
 type MarkdownMention = Pick<User, 'id' | 'username'>;
 
+export const mentionSpecialCharacters = new RegExp('[^a-zA-Z0-9_@]', 'g');
+
+type ReplacedCharacters = string[];
+
+const getReplacedCharacters = (word: string): [string, ReplacedCharacters] => {
```

Comment:
I suggest adding here a short comment explaining about the logic of this function